### PR TITLE
Fix some "current metrics" design issues

### PR DESCRIPTION
**Pull request by martinpitt**
First three issues in #31 

image

Overflow tooltip:

service-tooltip

### CODE REVIEW

File: src/app.scss [hunk 1]
```diff
@@ -7,6 +7,12 @@
   --card-width: 22rem;
   --pf-l-gallery--GridTemplateColumns: repeat(auto-fill, minmax(var(--card-width), 1fr));
 
+  // the cards already have padding, align the "top 5" tables with the other content
+  .pf-c-table {
+      --pf-c-table--cell--first-last-child--PaddingLeft: 0;
+      --pf-c-table--m-compact--cell--first-last-child--PaddingLeft: 0;
```
Comment (martinpitt) [inline src/app.scss:13]: Wait, I didn't mean to set both of them, that's a leftover from experimentation. It doesn't hurt, but one should suffice.

Comment (martinpitt) [reply src/app.scss]: Fixed.

**Comment by martinpitt** [comment]
We should probably make the value column right justified?

**Comment by martinpitt** [comment]
> We should probably make the value column right justified?

We don't seem to do that anywhere else in cockpit, and the [Table component](https://www.patternfly.org/v4/documentation/react/components/table) can't do that anyway, just left (default) or centered. So there's probably a reason for that :wink:

File: src/app.scss [hunk 2]
```diff
@@ -7,6 +7,12 @@
   --card-width: 22rem;
   --pf-l-gallery--GridTemplateColumns: repeat(auto-fill, minmax(var(--card-width), 1fr));
 
+  // the cards already have padding, align the "top 5" tables with the other content
+  .pf-c-table {
```
Comment (KKoukiou) [inline src/app.scss:11]: So visually this is correct but it's slightly inconsistent with the rest of cockpit. In all new pages that I re-factored with cards the previous days, we use `contains-list` class from lib/ct-card.scss for cards that contain tables. That removes completely the padding from the CardBody.

I prefer design wise the version you have here, I think the small paddings that come from the card body are nice to have. But I would also like that we have consistency.

Not a blocker for this PR, just raising the point.

**Review by KKoukiou — APPROVED** [review]

File: src/app.scss (hunk 2 above)
Comment (martinpitt) [reply src/app.scss:11]: FTR, from IRC: I'd actually prefer having the padding in the cards. That's easier than making sure that Table, Progress, DescriptionList, and anything else that we want to put in there has the same horizontal padding.

**Pull request merged**